app/views/mentoring/automation/tooltip_locked.html.haml

```diff
@@ -1,21 +1,20 @@
-.heading.c-generic-tooltip.text-14.w-413
-  %h4.font-semibold.mx-24.mt-20.mb-4 What is automation?
-
-  -# %p.mx-24
-    = ReactComponent::
-    Requires a satisfaction percentage of
-    %strong= @min_satisfaction_percentage.to_s
-    (current: #{@satisfaction_percentage})
-
-  -# %p.mx-24
+.p.heading.c-generic-tooltip.text-14.some{style:'padding:16px 24px;width:413px;'}
+  %p.text-16.text-white.font-semibold.leading-170.mb-12
     Requires
-    %strong= @min_finished_mentoring_sessions.to_s
-    finished mentoring sessions
-    (current: #{@finished_mentoring_sessions})
+    %strong.font-bold= @min_finished_mentoring_sessions.to_s
```

Comment:
```suggestion
    %strong.font-bold= @min_finished_mentoring_sessions
```

No need for `to_s` here I don't think.

Reply:
OK. and how does this work, isn't it a number? or when do I need to stringify it?

Reply:
Great question. So when something is rendered as a string in ruby, `to_s` is automatically called on it. So when something is interpolated (e.g. `width:#{num}%`}, num has `to_s` called on it. And when it's rendered here in haml, that happens too.